packages/squiggle-lang/src/value/index.ts

```diff
@@ -279,6 +279,23 @@ export type Plot =
       ids: string[];
     };
 
+export type TableChart = {
```

Comment:
Huh, I didn't expect that you'll rename this to `TableChart` too, `Table` here would be enough, I think.

Our previous pattern was to have `FooChart` in React components, and `Plot` or other types (e.g. `Lambda`) in squiggle-lang.

(I don't care about this much either way, though)